packages/esm-ohri-pmtct-app/src/form-entry/forms/labour-and-delivery/1.0.json

```diff
@@ -239,7 +239,7 @@
                   "intent":"*",
                   "required":"true",
                   "hide":{
-                    "hideWhenExpression":"client_booked !== '1065AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA'"
+                    "hideWhenExpression":""
```

Comment:
does this mean there's no need for any hide anymore?

Reply:
@pirupius Yeap, ideally the field isn't supposed to hide. No Business Rule needed it to. The assumption is the mother has not done an ANC and thus HIV status is a required question since they have chosen to do an L&D instead.